ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/user_management_service.proto

```diff
@@ -160,6 +180,27 @@ message GetUserResponse {
     User user = 1;
 }
 
+// Required authorization: ``HasRight(ParticipantAdmin)``
+// NOTE: The authenticated participant user is not allowed to deactivate itself.
+message UpdateUserRequest {
+    // The user to update.
+    // Modifiable fields:
+    // 1. 'primary_party',
+    // 2. 'is_deactivated',
+    // 3. 'metadata' (can be nested, e.g. 'metadata.annotations').
```

Comment:
Consider moving that to the messages themselves, as otherwise it easily goes out of data; and we end up repeating info like `metadata.annotations`.

Reply:
Yes. That was my intent behind adding the `Read-Only` and `Read-Write` markers.

Reply:
Ok, done